platform-server-kernel/src/main/java/uno/cod/platform/server/core/domain/Template.java

```diff
@@ -0,0 +1,45 @@
+package uno.cod.platform.server.core.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Template extends IdentifiableEntity implements StoredObject{
+    public static final String BUCKET = "coduno-templates";
```

Comment:
this could be "coduno-template-bucket" and should be configurable, i'm also unsure if we need different buckets for everything, or if we just simply the thing and use one bucket for the platform